addons/bus/static/src/js/services/bus_service.js

```diff
@@ -4,13 +4,46 @@ odoo.define('bus.BusService', function (require) {
 var CrossTab = require('bus.CrossTab');
 var core = require('web.core');
 var ServicesMixin = require('web.ServicesMixin');
+const session = require('web.session');
 
 var BusService =  CrossTab.extend(ServicesMixin, {
     dependencies : ['local_storage'],
 
     // properties
     _audio: null,
 
+    /**
+     * As the BusService doesn't extend AbstractService, we have to replicate
```

Comment:
Probably out of scope, but why is it not the case, or couldn't it be made so?

Reply:
Probably because BusService already extends CrossTab, so AbstractService (or CrossTab) should be a mixin...

Reply:
I see, well that's out of scope then.